ONSdigital/eq-author
# Add/remove new items

Adds the functionality for adding and removing Sections/Questions/Answers

## How to test

Create an empty survey and check you can add multiple sections/questions and answers
